### PR TITLE
README update to be aligned with latest changes in set up procedure

### PR DESCRIPTION
**Pull request by shtukas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,9 @@ The process is as follows:
 
 1. An ETL step downloads recipe articles from the Content API, makes a best-effort pass to extract individual recipes and parse them, and writes them to a PostgreSQL DB.
 
-2. A Play app selects semi-parsed recipes from the DB, displays them to users and guides them through the process of parsing/verifying them.
+2. You will then need to call an end point that migrates `New` recipes (the outcome of the previous point) into `Ready` status. (See section "Running ETL locally").
```
Comment (LATaylor-guardian) [inline README.md]: Why do we need to do this now?

File: README.md [hunk 2]
```diff
@@ -25,6 +27,12 @@ $ sbt "etl/run <capi key>"
 
 This should run for a few minutes. By the end, you will have a few thousand recipes in your DB.
 
+The next step is then to call the `/admin/prepare-recipes` end point which migrates recipes in `New` status into `Ready`. Note that you need have the play app running to do this. (See "Running play app")
```
Comment (shtukas) [inline README.md]: This is called by me, as part of the manual step by step migration of recipes from `New` to `Ready`. The operation is meant to be idempotent.

File: README.md (hunk 1 above)
Comment (shtukas) [inline README.md]: Just keeping the README up to date with the latest version of the code

File: README.md (hunk 2 above)
Comment (shtukas) [inline README.md]: This is the code as it currently is, meaning that if somebody wants to install the system from scratch locally *today* they will have to do that. I will keep the README up to date as the code and set up process evolves :)

**Comment by tomrf1** [comment]
👍

**Pull request merged**